packages/app-mobile/tsconfig.json

```
@@ -13,6 +13,7 @@
 		"tools/*.ts",
 	],
 	"compilerOptions": {
-		"types": ["jest", "node"]
+		"types":["jest", "node", "react-test-renderer"],
```

```suggestion
		"types": ["jest", "node", "react-test-renderer"],
```
(very, very small change)

---

Thanks!